### ISSUE TITLE
Feature Request BirdNet Lite

### ISSUE THREAD

**Issue by HarshitSheoran** [PINTO0309/PINTO_model_zoo#338]
### Issue Type

Feature Request

### OS

Ubuntu

### OS architecture

x86_64

### Programming Language

Python

### Framework

TensorFlow

### Model name and Weights/Checkpoints URL

https://github.com/PINTO0309/PINTO_model_zoo/tree/main/177_BirdNET-Lite

### Description

I want this model in keras, the reason for that is, I want to remove the preprocessing part which have RFFT layer, and take outputs at both embedding and logits, and I am unable to do this with tensorflow graph

**Comment by PINTO0309** [comment]
I thought if you could convert from TensorFlow saved_model to ONNX it would be very easy to achieve what you want. Unfortunately, however, it seems that tensorflow-onnx aborts on the `RFFT2D` conversion.

If you really want to break up the model, you will need to convert the tflite to JSON using the tool below, manually cut out the before and after RFFT2D, and then generate the tflite or Keras, saved_model from the JSON again.

https://github.com/PINTO0309/tflite2json2tflite

I know this is a very difficult task because you need to understand the FlatBuffer specifications, but good luck.

**Comment by HarshitSheoran** [comment]
![image](https://github.com/PINTO0309/PINTO_model_zoo/assets/37008327/ddee1b6c-0f4b-49a1-aa04-904b3f62887d)
I don't think this error provides much detail on why

**Comment by HarshitSheoran** [comment]
Solved that error, apprently not using sudo can be a dumb thing to do while using docker

**Comment by HarshitSheoran** [comment]
I have tried some things, I have no idea how to delete the part of graph, it looks simple in netron but the changes seems complex in .json that was created, do you have tips for me?

**Comment by PINTO0309** [comment]
Since you are still trying to figure out what is best for you, I will share as many tutorials and simple steps as I can think of. It is not in my or the community's best interest to address individual requirements, and I am not a teacher, so I will keep the explanation and work costs to a minimum. Please understand.

- My article (Unfortunately, it is all in Japanese.)
  [How to regenerate tflite after converting tflite with Custom Operation to JSON and replacing it with standard OP](https://zenn.dev/pinto0309/articles/9d316860f8d418)

Alternatively, it would be far easier to increase the final output to two or three rather than go to the trouble of removing all the intermediate structure of the model.

- Before
  [Acos_11_float32.tflite.zip](https://github.com/PINTO0309/PINTO_model_zoo/files/11475525/Acos_11_float32.tflite.zip)
  ![image](https://github.com/PINTO0309/PINTO_model_zoo/assets/33194443/b4b3f767-8296-429a-b7ac-779b3a92804c)
  ![image](https://github.com/PINTO0309/PINTO_model_zoo/assets/33194443/f5f67607-1e7b-483b-8004-f4f9c6ec0212)
- After
  [Acos_11_float32.tflite.zip](https://github.com/PINTO0309/PINTO_model_zoo/files/11475575/Acos_11_float32.tflite.zip)
  ![image](https://github.com/PINTO0309/PINTO_model_zoo/assets/33194443/85e30f79-f93c-4663-bd56-5c7f5a63e309)
  ![image](https://github.com/PINTO0309/PINTO_model_zoo/assets/33194443/832ab749-6971-4167-a4fa-af72e2554e83)

**Comment by PINTO0309** [comment]
I'm going to close it since you don't seem to want to take it seriously.
